lib/linguist/generated.rb

```diff
@@ -533,5 +534,14 @@ def generated_grpc_cpp?
       return false unless lines.count > 1
       return lines[0].start_with?("// Generated by the gRPC")
     end
+
+    # Internal: Is the file a generated Perl/Pollution/Portability header file?
+    #
+    # Returns true or false.
+    def generated_perl_ppport_header?
+        return false unless name.match(/ppport\.h$/)
+        return false unless lines.count > 10
```

Comment:
Why 10? Is that arbitrary?

Reply:
The string match in the next test is on line 9, so it's just to make sure the file is at least that long.

Reply:
Ah, ok. I guess I was expecting `> 9` or a higher number for the whole header comment.